Sources/Account/Account+TransactionOperations.swift

```diff
@@ -115,7 +132,7 @@ extension Account {
                 return
             }
 
-            feeFetcher.feeStrategy(for: feeLevel) {
+            metaFetcher.feeStrategy(for: feeLevel) {
```

Comment:
`metaFetcher.feeStragety` looks like it'll just use the cached value, if present... can it change or maybe not be fully up to date? 

Reason I'm asking is b/c metaFetcher.blockVersion` can trigger a `fetchMeta` which looks like it may also udpate feeStrategy...  

So seems like the fee from this call could be stale but used anyways.

If this is a problem, maybe we can flip the order of the calls (check block then get fee), so that the block number is checked first - which will update the fee in tandem, then the subsequent call to get the fee will be using the current value pulled by the fetchMeta call.

Reply:
this change is rename so don't think it makes sense to refactor behavior if its already working and tested. Answering your question: feeStrategy() { ... }  has a closure so that it uses the cached value if its not-expired, and fetches the network if it is, then executes the closure.

Reply:
Ah, so I'm not referring to the rename...  I'm referring to the addition of the block check, where the block check will actually refresh the value, but the fee check pulls the cached value.   I could have put the note further down, but I wanted to associate it with the location where the fee was being pulled.   Really it's regarding the order of block versus fee.

Reply:
I think I understand your comment now, so if the cache is stale the feeStrategy() call will update it, and then the subsequent blockVersion call will essentially always pull from the cache cus it just got updated by the feeStrategy call. I was thinking to make a call to combine both into one function to avoid the double scope nesting but didn't want to add more overhead to this change and figured the extra scope was an ok tradeoff for the time saved implementing.